gslb/nodes/avi_model_nodes.go

```diff
@@ -117,28 +121,39 @@ func (gsk8sObj AviGSK8sObj) getCopy() AviGSK8sObj {
 	return obj
 }
 
+// This struct is used to store all details for path based Hms
+type HealthMonitorPathDescription struct {
+	HmName      string
+	GsName      string
+	Path        string
+	Protocol    string // http or https
+	Description string
```

Comment:
As suggested, please check if we can use a struct here instead of a string for `Description`.